### PR TITLE
fix(overthebox): resolve empty guides for some languages

### PR DESCRIPTION
**Pull request by Steffy29**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #UXCT-44, UXCT-288
| License          | BSD 3-Clause

## Description

Resolve empty guides for some languages like DE, PL, PT, FI, ES...
Fix typo on Android label
Refactoring the code for over the box parts

### CODE REVIEW

File: packages/manager/modules/overthebox/src/overthebox/overTheBox.routing.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    abstract: true,
+    component: 'ovhManagerOverTheBoxComponent',
+    resolve: {
+      service: /* @ngInject */ (OvhApiOverTheBox, $stateParams) =>
+        OvhApiOverTheBox.v6()
+          .get({ serviceName: $stateParams.serviceName })
+          .$promise.then((service) => service),
+      serviceName: /* @ngInject */ ($transition$) =>
+        $transition$.params().serviceName,
+      $title(translations, $translate, $stateParams, OvhApiOverTheBox) {
```
Comment (cbourgois) [inline packages/manager/modules/overthebox/src/overthebox/overTheBox.routing.js:13]: Please use explicit annotation: 
```suggestion
      $title: /* @ngInject */ (translations, $translate, $stateParams, OvhApiOverTheBox) => {
```

**Comment by Steffy29** [comment]
@cbourgois or perhaps a rebase a little quickly.

**Comment by Steffy29** [comment]
@antleblanc resolved

**Comment by ovh-cds** [comment]
CDS Report build#13726.0 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build#13726.1 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build#13726.2 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build#13729.0 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build#13731.0 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build#13732.0 ✘  
* 
  * Build ✘

File: packages/manager/modules/overthebox/src/overthebox/overTheBox.component.js [hunk 2]
```diff
@@ -1,91 +1,12 @@
...
-    controllerAs: 'OverTheBox',
-  })
-  .constant('OVER_THE_BOX', constant)
-  .constant('PAGINATION_PER_PAGE', 5);
-
-export default moduleName;
+export default {
+  controller,
+  template,
+  controllerAs: 'OverTheBox',
```
[the remark this answers is not in the document]
Comment (Steffy29) [reply packages/manager/modules/overthebox/src/overthebox/overTheBox.component.js:7]: To have less work to do, I note this for a future update

File: packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.controller.js [hunk 3]
```diff
@@ -67,7 +121,8 @@ export default /* @ngInject */ function(
    * @param series
    * @returns {{max: number, current: number, rateMbps: number, rateUnit: string}}
    */
-  function computeSpeed(series) {
+  // eslint-disable-next-line class-methods-use-this
+  computeSpeed(series) {
```
[the remark this answers is not in the document]
Comment (Steffy29) [reply packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.controller.js:125]: I would like to do like you said but if I do this, I have errors "this.computeSpeed is not a function"

File: packages/manager/modules/overthebox/src/overthebox/configure/overTheBox-configure.controller.js [hunk 4]
```diff
@@ -1,32 +1,27 @@
...
+    this.$state = $state;
+    this.OvhApiOverTheBox = OvhApiOverTheBox;
+    this.TucToastError = TucToastError;
+  }
 
-  function init() {
-    self.loading = true;
-    return OvhApiOverTheBox.v6()
+  $onInit() {
+    this.orderHash = this.$state.href('order-overTheBox');
```
[the remark this answers is not in the document]
Comment (Steffy29) [reply packages/manager/modules/overthebox/src/overthebox/configure/overTheBox-configure.controller.js:11]: I note this for a previous task.

File: packages/manager/modules/overthebox/src/overthebox/configure/overTheBox-configure.routing.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+export default /* @ngInject */ ($stateProvider, $urlRouterProvider) => {
+  $stateProvider.state('overTheBox-configure', {
```
[the remark this answers is not in the document]
Comment (Steffy29) [reply packages/manager/modules/overthebox/src/overthebox/configure/overTheBox-configure.routing.js:2]: I note this for a previous task

File: packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.routing.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('overTheBoxes.overTheBox.details', {
+    url: '/details',
+    views: {
+      otbView: 'overTheBoxDetails',
+    },
+    translations: {
+      value: ['.', '../remote'],
```
[the remark this answers is not in the document]
Comment (Steffy29) [reply packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.routing.js:8]: I note this for a previous task

File: packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.controller.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (JeremyDec) [reply packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.controller.js:125]: @Steffy29 you have to write `this.constructor.computeSpeed` to access static method

**Pull request merged**